main.nf

```diff
@@ -2945,18 +2966,25 @@ process Mpileup {
         file(fastaFai) from ch_fai
 
     output:
-        set idPatient, idSample, file("${prefix}${idSample}.pileup.gz") into mpileupMerge
+        set idPatient, idSample, file("${prefix}${idSample}.pileup") into mpileupMerge
 
     when: 'controlfreec' in tools || 'mpileup' in tools
 
     script:
     prefix = params.no_intervals ? "" : "${intervalBed.baseName}_"
     intervalsOptions = params.no_intervals ? "" : "-l ${intervalBed}"
+
+    if(params.normal_pileup && params.tumor_pileup)   // we already have pileups
+    """
+    touch ${prefix}${idSample}.pileup 
+    """
+    else    
     """
+    # Control-FREEC reads uncompresses the zipped file TWICE in single-threaded mode.
+    # For the moment we are not using compressed pileups.
     samtools mpileup \
         -f ${fasta} ${bam} \
-        ${intervalsOptions} \
-    | bgzip --threads ${task.cpus} -c > ${prefix}${idSample}.pileup.gz
+        ${intervalsOptions} > ${prefix}${idSample}.pileup # | bgzip --threads ${task.cpus} -c > ${prefix}${idSample}.pileup.gz
```

Comment:
if the output is not with the `.gz` I don't think we're saving it in the results folder actually.

Reply:
You are right, have to be fixed in the publishDir as I see